pyproject.toml

```diff
@@ -1,3 +1,92 @@
+# scriv's pyproject.toml
+
+[project]
+name = "scriv"
+description = "Scriv changlog management tool"
+authors = [
+    {name = "Ned Batchelder", email = "ned@nedbatchelder.com"},
+]
+license.text = "Apache-2.0"
```

Comment:
As of two days ago, `setuptools` supports PEP 639. It's a good time to switch to the new SPDX identifier.
* https://packaging.python.org/en/latest/guides/licensing-examples-and-user-scenarios/#licensing-examples (the doc still says that setuptools doesn't support it, but it does).
* https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license
* https://packaging.python.org/en/latest/specifications/pyproject-toml/#license

Reply:
Did you see the errors I was getting?

Reply:
@nedbat I only noticed them later. But now I concluded that you likely need to invalidate cache: https://github.com/nedbat/scriv/pull/144#issuecomment-2743263076.